docs/resources/iotda_device_certificate.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "IoT Device Access (IoTDA)"
+---
+
+# huaweicloud_iotda_device_certificate
+
+Manages an IoTDA device CA certificate within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "certificateContent" {}
+
+resource "huaweicloud_iotda_device_certificate" "test" {
+  content  = var.certificateContent
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the IoTDA device CA certificate
+resource. If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `content` - (Required, String, ForceNew) Specifies the content of device CA certificate.
+Changing this parameter will create a new resource. Create a private CA certificate,
+please following [reference](https://support.huaweicloud.com/usermanual-iothub/iot_01_0104.html)
+
+* `space_id` - (Optional, String, ForceNew) Specifies the resource space ID to which the device CA certificate belongs.
+If omitted, the certificate will belong to the default resource space.
+Changing this parameter will create a new resource.
+
+* `verify_content` - (Optional, String) Specifies the content of verification certificate. Used to verify the validity
+of the device CA certificate. Get the verification certificate,
+please following [reference](https://support.huaweicloud.com/usermanual-iothub/iot_01_0106.html)
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID in UUID format.
```

Comment:
seems it's not UUID format as the import example at L63

Reply:
fix